src/ECER.Tests/ECER.Tests.csproj

```diff
@@ -31,4 +31,8 @@
     <ProjectReference Include="..\ECER.Resources.Accounts\ECER.Resources.Accounts.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="Integration\Managers\Registry\" />
```

Comment:
missing tests?

Reply:
No, Tests Covered in PortalInvitationTests

Reply:
then probably revert this added folder